pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3830,11 +3830,13 @@
       "BaselineVersion": "4.3.0",
       "InboxOn": {
         "netcoreapp2.0": "4.0.2.0",
-        "netcoreapp2.1": "4.0.3.0"
+        "netcoreapp2.1": "4.0.3.0",
+        "netcoreapp3.0": "4.1.0.0"
       },
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
-        "4.0.1.0": "4.3.0"
+        "4.0.1.0": "4.3.0",
+        "4.1.0.0": "4.6.0"
```

Comment:
@ericstj I am not sure if the 4.6.0 is correct here, I was guessing.

Reply:
Yes, this is correct per https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/Packaging.props#L36.  It's also what we auto-commit if you run the /t:UpdatePackageIndex on the pkgproj.